book/src/csi-proxy.md

```diff
@@ -0,0 +1,44 @@
+# CSI Proxy
+
+## Status and Releases
+
+**Git Repository:** [https://github.com/kubernetes-csi/csi-proxy](https://github.com/kubernetes-csi/csi-proxy)
+
+**Status:** Beta starting with v0.2.0
+
+Status | Min K8s Version | Max K8s Version
+--|--|--
+Alpha | 1.18 | - 
+Beta | 1.18  | -
+
+## Description
+
+CSI Proxy is a binary that exposes a set of gRPC APIs around storage operations over named pipes in Windows. A container, such as CSI node plugins, can mount the named pipes depending on operations it wants to exercise on the host and invoke the APIs.
+
+Each named pipe will support a specific version of an API (e.g. v1alpha1, v2beta1) that targets a specific area of storage (e.g. disk, volume, file, SMB, iSCSI).
+
+## Usage
+
+Run csi-proxy.exe binary directly on Window node. Command line options
+
+* `-kubelet-csi-plugins-path`: This is the prefix path of the Kubelet plugin directory in the host file system (`C:\var\lib\kubelet` is used by default).
```

Comment:
this path is mainly for validating the path requested from client is valid (under pod volume dir), not some other arbitrary path for security reason.

Reply:
Can you add more details of what this path is used for in the description of this field? The name of this field is also confusing because it's the kubelet root, not the plugins path.

Reply:
we are in the process of hanging this part to use kublet-path instead of two to match the name.  will update it after the change is done.

Reply:
has this been updated for v1?

Reply:
`-kubelet-csi-plugins-path` and `-kubelet-pod-path` below should be replaced with (or noted as removed in favor of) `-kubelet-pod-path` in v1.0.

Reply:
updated, thanks!